groovy/gradle/src/org/netbeans/modules/gradle/execute/GradleDaemonExecutor.java

```diff
@@ -156,6 +158,9 @@ public void run() {
             if (!activePlatform.second().getInstallFolders().isEmpty()) {
                 File javaHome = FileUtil.toFile(activePlatform.second().getInstallFolders().iterator().next());
                 buildLauncher.setJavaHome(javaHome);
+                Map<String, String> envs = new HashMap<>(System.getenv());
+                envs.put("JAVA_HOME", javaHome.getCanonicalPath());
```

Comment:
This can throw an IOException, which is not yet caught and thus breaks the build.

Reply:
Yes, that's true. Thank you for the check! It shall be fixed now.